posts/01-first-map/tjstogoski/brnp-map.md

```diff
@@ -0,0 +1,5 @@
+# Banshee Reeks Nature Preserve, the hidden treasure of Northern Virginia
+
+Banshee Reeks Nature Preserve (BRNP) protects 695 acres of the Northern Virginia Piedmont through a conservation easement held by the Virginia Outdoors Foundation. It is managed by the Department of Parks, Recreation & Community Services of Loudoun County, Virginia, USA. The map shows the forests, fields, wetlands, and waters that make up the preserve and serve to teach us of the natural and cultural resources of the region.
```

Comment:
This is a great description of the area. This works for the assignment @nquarder

Reply:
Yes agreed, @VictoriAragon - @tjstogoski when you get a second try adding a hyperlink to BRNP or another citation of some kind.